guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -372,7 +369,11 @@ stage:
 -  **Default**—`true`
 -  **Version**—Magento 2.1.4 and later
 
-Generates symlinks for static content. This setting is vital in the Pro Production environment for the three-node cluster. When this variable is set to `false`, it must copy every file during the deployment, which increases deployment time. Setting `SCD_ON_DEMAND` to `true` disables this variable.
+Generates symlinks for static content. This setting is vital in the Pro Production environment for the three-node cluster. When this variable is set to `false`, it must copy every file during the deployment, which increases deployment time.
```

Comment:
it must copy?  What --  I think sometimes the passive is clearer... every file must be copied during the deployment .... 

or the deployment process must copy every file, which increases ...